clang/lib/Frontend/VerifyDiagnosticConsumer.cpp

```diff
@@ -1098,7 +1098,14 @@ void VerifyDiagnosticConsumer::CheckDiagnostics() {
     // Produce an error if no expected-* directives could be found in the
     // source file(s) processed.
     if (Status == HasNoDirectives) {
-      Diags.Report(diag::err_verify_no_directives).setForceEmit();
+      std::string Err_Directive;
+      if (Diags.getDiagnosticOptions().VerifyPrefixes.empty()) {
+        Err_Directive = "expected";
+      } else {
+        Err_Directive = *Diags.getDiagnosticOptions().VerifyPrefixes.begin();
+      }
+      Diags.Report(diag::err_verify_no_directives).setForceEmit()
+          << Err_Directive;
```

Comment:
```suggestion
      std::string ErrDirective;
      if (Diags.getDiagnosticOptions().VerifyPrefixes.empty())
        ErrDirective = "expected";
      else
        ErrDirective = *Diags.getDiagnosticOptions().VerifyPrefixes.begin();
      Diags.Report(diag::err_verify_no_directives).setForceEmit()
          << ErrDirective;
```
We might want to split this logic out into a helper function if we're also updating `err_verify_invalid_no_diags` at the same time.

Reply:
Please let me know if we are doing this also in this PR. I think its best to raise a separate issue for the same as LLVM PR standards imply.